components/script/dom/htmllinkelement.rs

```diff
@@ -357,10 +357,21 @@ impl HTMLLinkElementMethods for HTMLLinkElement {
     make_setter!(SetHref, "href");
 
     // https://html.spec.whatwg.org/multipage/#dom-link-rel
-    make_getter!(Rel, "rel");
+    fn Rel(&self) -> DOMString {
+        let atomVec: Vec<Atom> = self.upcast::<Element>().get_tokenlist_attribute(&atom!("rel"));
+        let mut string_concat = "".to_string();
+        for data in &atomVec {
+            let dataStr = String::from(data as &str);
+            string_concat = string_concat + " " + &dataStr;
+        }
```

Comment:
Instead of doing this in imperative style, I think you can do something fancy with iterators:
```rust
DOMString::from(atom_vec.iter().map(|a| &*a).join(" "))
```